packages/airnode-starter/integrations/coingecko-testable/create-secrets.ts

```diff
@@ -0,0 +1,25 @@
+import { writeFileSync } from 'fs';
+import { join } from 'path';
+import { randomUUID } from 'crypto';
+import { ethers } from 'ethers';
+import { formatSecrets, getDeployedContract, readChainId, readIntegrationInfo } from '../../src';
+
+async function createSecrets() {
+  const integrationInfo = readIntegrationInfo();
+  const wallet = ethers.Wallet.createRandom();
+  const airnodeRrp = await getDeployedContract('@api3/protocol/contracts/rrp/AirnodeRrp.sol');
+
+  const secrets = [
+    `PROVIDER_URL=${integrationInfo.providerUrl}`,
+    `AIRNODE_WALLET_MNEMONIC=${wallet.mnemonic.phrase}`,
+    `HTTP_GATEWAY_API_KEY=${randomUUID()}`,
+    `AIRNODE_RRP_ADDRESS=${airnodeRrp.address}`,
+    `CHAIN_ID=${await readChainId()}`,
+    `CLOUD_PROVIDER_TYPE=${integrationInfo.airnodeType}`,
+  ];
+
+  writeFileSync(join(__dirname, 'secrets.env'), formatSecrets(secrets));
```

Comment:
Good point. We definitely should add a command to clean remove the Airnode from AWS and also remove the container (when running locally). That command could remove the `secrets.env` file as well.

And we should also have a script to clean all generated files (just in case).

Reply:
Actually I thought about this more and I would just overwrite - the worst that will happen is that you will overwrite the secrets of currently used Airnode. But this is not a big issue sine this package is only for examples - you should not use this package for your main app integration. For that we should have dedicated blog posts or separate README saying what is necessary.

I also do not want to add too much warnings and blocking conditions, because technically there is nothing bad with overwriting the secrets.env (the use case is you want to deploy another Airnode with different mnemonic).

Reply:
Sure, just wanted to check that this would not be an issue